docs/changelog.md

```diff
@@ -0,0 +1,88 @@
+---
+id: Changelog
+toc_min_heading_level: 2
+toc_max_heading_level: 2
+---
+
+# Changelog
+
+## October 6, 2023
+
+### Breaking Changes (Code Incompatibility)
+
+❗[responsive-pub/134](https://github.com/responsivedev/responsive-pub/pull/134) 
+and [responsive-pub/135](https://github.com/responsivedev/responsive-pub/pull/135)
+changed the way a new instance of `ResponsiveKafkaStreams` is created. These
+changes apply to `dev.responsive.kafka-client` versions `0.12.0` and above.
+
+Please modify your code to use the constructor place of the static factory 
+method:
+
+```diff showLineNumbers
+-  KafkaStreams streams = ResponsiveKafkaStreams.create(...);
++  KafkaStreams streams = new ResponsiveKafkaStreams(...);
+```
+
+It also removed the timestamped verisons of `StoreSuppliers` since all Responsive
+stores are timestamped. You can apply all of these changes with the following
+`sed` commands:
+
+```bash
+sed -ie "s|ResponsiveKafkaStreams.create|new ResponsiveKafkaStreams|g" ${FILE}
+sed -ie "s|ResponsiveKeyValueParams.timestamped|ResponsiveKeyValueParams.keyValue|g" ${FILE}
+sed -ie "s|ResponsiveKeyValueParams.timestampedFact|ResponsiveKeyValueParams.fact|g" ${FILE}
+sed -ie "s|ResponsiveStores.timestampedKeyValueStore|ResponsiveStores.keyValueStore|g" ${FILE}
+```
+
+❗[responsive-pub/148](https://github.com/responsivedev/responsive-pub/pull/148):
+This change restructured code so to distinguish between public apis and
+internal implementation classes. Some classes, therefore, may have slightly
+shifted package names. If you fail to compile after upgrading versions,
+remove the imports that are not found and re-import them with the same class
+name.
+
+
+### Operator (0.10.0)
+
+✨[responsive-pub/120](https://github.com/responsivedev/responsive-pub/pull/120):
+[Expected Latency](https://docs.responsive.dev/reference/controller-policies#expected-latency)
+diagnoser was introduced that allows you to specify a goal for the expected 
+latency for a record to be processed at a sub-topology (how long it takes
+from the point it is enqueud on to the topic to when it has been processed).
+
+✨[responsive-pub/131](https://github.com/responsivedev/responsive-pub/pull/131):
+Introduced a [Thread Saturation Diagnoser](https://docs.responsive.dev/reference/controller-policies#thread-saturation), 
+which will scale down the number of nodes in your application if threads are not 
+being fully utilized.
+
+### Kafka Client (0.12.0)
+
+✨[responsive-pub/133](https://github.com/responsivedev/responsive-pub/pull/133):
+Support configuring changelog truncation. This feature will allow you to 
+optionally truncate changelogs when using Responsive stores to reduce storage
+utilization on your Kafka cluster. We recommend leaving it enabled as an
+additional source of data recovery.
```

Comment:
This is confusing -- from the previous sentences it seems like the "it" should be "changelog truncation", but you're actually trying to say the opposite here and I gather that the "it" is actually intended to be "the changelog" itself. 

Also, the "as an additional source of data recovery." kind of makes it sound like we'd be truncating records right away and potentially losing data if the node goes down -- not exactly sure how to resolve this, maybe just add a sentence or two before the last one that clarifies we are only truncating data that has already been persisted to the Responsive storage engine? Now that I think of it, that's probably a good idea to mention here regardless of the "additional source of data recovery" thing

```suggestion
utilization on your Kafka cluster. We recommend leaving it disabled and keeping your changelogs intact as an
additional source of data recovery.
```